coreneuron/nrniv/main1.cpp

```diff
@@ -437,9 +442,7 @@ extern "C" int run_solve_core(int argc, char** argv) {
         double tstop = nrnopt_get_dbl("--tstop");
 
         if (tstop < t && nrnmpi_myid == 0) {
-            printf("Error: Stop time (%lf) < Start time (%lf), restoring from checkpoint? \n",
-                   tstop, t);
-            abort();
+            spdlog::error("Error: Stop time (%lf) < Start time (%lf), restoring from checkpoint? {} {}", tstop, t);
```

Comment:
@tapaswenipathak : if I am not mistaken, `spdlog::error` just print message but doesn't abort?

Reply:
Yea, you can add an exit code.